samples/MAUI/LibVLCSharp.MAUI.Sample.MediaElement/Platforms/Tizen/Main.cs

```diff
@@ -0,0 +1,17 @@
+using System;
```

Comment:
tizen folder should be removed to avoid confusion I think. we don't support tizen.

Reply:
Done